keystone.ts

```diff
@@ -14,6 +16,29 @@ export default withAuth(
         origin: "*",
         methods: "GET,HEAD,PUT,PATCH,POST,DELETE",
       },
+      extendExpressApp: (app, commonContext) => {
```

Comment:
nit: we can remove `commonContext`, this is used if we wanted to interact with the GraphQL db here which we don't need for this endpoint.